oxidation/crates/parsec_api_types/src/ext_types.rs

```diff
@@ -140,6 +141,13 @@ impl<'de> serde::Deserialize<'de> for DateTime {
     }
 }
 
+impl Sub for DateTime {
+    type Output = Duration;
```

Comment:
We should re-expose `chrono::Duration` then (the idea would be to never rely on chrono outside of here)

Reply:
Done :+1: